### PR TITLE
BUG: take only the first referenced RWVM series

### PR DESCRIPTION
**Pull request by fedorov**

### CODE REVIEW

File: Py/DICOMTID1500Plugin.py [hunk 1]
```diff
@@ -124,24 +124,24 @@ def load(self, loadable):
...
+          if len(rwvmFiles)>0:
             # consider only the first item on the list - there should be only
             # one anyway, for the cases we are handling at the moment
             rwvmFile = rwvmFiles[0]
             logging.debug("Reading RWVM from "+rwvmFile)
             rwvmDataset = dicom.read_file(rwvmFile)
-            if hasattr(rwvmDataset,"ReferencedSeriesSequence"):              
+            if hasattr(rwvmDataset,"ReferencedSeriesSequence"):
               if hasattr(rwvmDataset.ReferencedSeriesSequence[0],"SeriesInstanceUID"):
                 if rwvmDataset.ReferencedSeriesSequence[0].SeriesInstanceUID == segLoadable.referencedSeriesUID:
```
Comment (che85) [inline Py/DICOMTID1500Plugin.py]: Can we be sure that segLoadable.referencedSeriesUID exists? You removed the check from line 127

**Comment by che85** [comment]
Other than that looks good to me

File: Py/DICOMTID1500Plugin.py (hunk 1 above)
Comment (fedorov) [inline Py/DICOMTID1500Plugin.py]: of course. I missed that. I fixed it and squashed/merge-pushed

**Comment by fedorov** [comment]
Due to a force push by mistake, the merge commit above was lost. This feature was committed in https://github.com/QIICR/QuantitativeReporting/commit/cca7b6e590c039ad181cd64042b4cfc02a242963

**Pull request merged**